daemon/stats.go

```diff
@@ -27,9 +26,11 @@ func (daemon *Daemon) ContainerStats(ctx context.Context, prefixOrName string, c
 		return errdefs.InvalidParameter(errors.New("cannot have stream=true and one-shot=true"))
 	}
 
+	outStream := config.OutStream()
```

Comment:
Nit (no need to change); I went looking where this was used; looks like we always construct a JSON encoder with this, so could move the `enc := json.NewEncoder(outStream)` from line 61 here;

```suggestion
	enc := json.NewEncoder(config.OutStream())
```

Reply:
Updated to do this.